ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/ValidatorsUtil.java

```diff
@@ -107,9 +107,7 @@ public static boolean is_eligible_for_activation(BeaconState state, Validator va
                       return i;
                     }
                   }
-                  // TODO would be nice to clean this up but it will be filtered out by the line
-                  // below
-                  return Integer.MAX_VALUE;
+                  return null;
```

Comment:
Looks good now 👍 
Just curious why it was `MAX_VALUE` before?

Reply:
I'm not really sure what I was thinking to be honest.